File: Java/iot_lab_with_tests/src/main/java/ua/lviv/iot/model/DefenceEquipment.java
package ua.lviv.iot.lab3.model;

import ua.lviv.iot.lab3.model.enums.EquipmentType;
import ua.lviv.iot.lab3.model.enums.WorkType;

public class DefenceEquipment extends Equipment {
    private String bodyPart;
    private int size;

public DefenceEquipment(final EquipmentType equipmentType, final WorkType workType, final double weight,
                        final String bodyPart, final int size) {
     this.setEquipmentType(equipmentType);
     this.setWorkType(workType);
     this.setWeight(weight);
     this.bodyPart = bodyPart;
     this.size = size;
    }

    @Override
public String toString() {
        return "DefenceEquipment{" + "weight=" + getWeight() + ", equipmentType=" + getEquipmentType() + ", workType="
                + getWorkType() + ", bodyPart='" + bodyPart  + '\'' + ", size=" + size + '}';
    }
}

File: Python/lab11/model/DefenceEquipment.py
from model.Equipment import Equipment


class DefenceEquipment(Equipment):
    def __init__(self, equipment_type, work_type, weight, body_part, size, id):
        self.equipment_type = equipment_type
        self.work_type = work_type
        self.weight = weight
        self.body_part = body_part
        self.size = size
        self.id = id

    def __init__(self, equipment_type, work_type, weight, body_part, size):
        self.equipment_type = equipment_type
        self.work_type = work_type
        self.weight = weight
        self.body_part = body_part
        self.size = size
        self.id = id